homeassistant/components/weatherflow/strings.json

```diff
@@ -14,11 +14,10 @@
     "error": {
       "address_in_use": "Unable to open local UDP port 50222.",
       "cannot_connect": "UDP discovery error.",
-      "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]"
+      "no_device_found": "[%key:common::config_flow::abort::no_devices_found%]"
     },
     "abort": {
```

Comment:
Doesn't this conflict with the work from @mib1185? I chatted with him and he said that there should rather be a key in the manifest instead of an abort message for `single_instance_allowed`.